servicetalk-dns-discovery-netty/docs/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,71 @@
+// Configure {source-root} values based on how this document is rendered: on GitHub or not
+ifdef::env-github[]
+:source-root:
+endif::[]
+ifndef::env-github[]
+ifndef::source-root[:source-root: https://github.com/apple/servicetalk/blob/{page-origin-refname}]
+endif::[]
+
+= Service Discovery
```

Comment:
It does not seem correct to have all docs related to Service Discovery under DNS as we may add more implementations for service discovery later. If you really want to extract Service Discovery into a separate doc then adding it to `client-api` would be more appropriate. 

I would suggest leaving this definition of `Service Discovery` in HTTP for now and add the doc describing DNS implementation here, optionally referring to the HTTP Service Discovery section in HTTP. On the other hand, if you want to move this definition under `client-api`, its fine by me.

Reply:
💯 `client-api` is a better place.